model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -195,6 +202,32 @@ class LicenseInfoResolver(
             ResolvedCopyright(statement, findings.toSet())
         }
     }
+
+    private fun createLicenseFileInfo(id: Identifier): ResolvedLicenseFileInfo {
+        val licenseInfo = resolveLicenseInfo(id)
+        val licenseFiles = mutableListOf<ResolvedLicenseFile>()
+        val matcher = FileMatcher(archiver.patterns)
+
+        licenseInfo.flatMapTo(mutableSetOf()) { resolvedLicense ->
+            resolvedLicense.locations.map { it.provenance }
+        }.forEach { provenance ->
+            val archiveDir = createTempDir(ORT_NAME, "notice").also { it.deleteOnExit() }
```

Comment:
Is "notice" a good name here, even if only used for a temporary directory? I mean yes, these license files are usually used in the context of `NOTICE` file generation, but we probably should not make that assumption here and use a more generic name like maybe just "licenses".

Reply:
Changed it to the very generic "archive".